src/foam/u2/wizard/DAOWizardlet.js

```diff
@@ -0,0 +1,43 @@
+foam.CLASS({
```

Comment:
copyright

Reply:
One day I'll submit a PR with all the license headers added. Maybe not today, but one day

Reply:
Fixed